Codidact.Authentication.sln

```diff
@@ -0,0 +1,116 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio 15
```

Comment:
I see you are using Visual Studio 2017 (I think?) wouldn't this need to be ported to 2019?

Reply:
I created this file using `dotnet new sln` from the command line. From the documentation:

  >     # Visual Studio Version 16
  > The major version of Visual Studio that (most recently) saved this solution file. This information controls the version number in the solution icon. &mdash;&nbsp;[source][1]

I do not think anything needs to be changed here, the important part seems to be

    Microsoft Visual Studio Solution File, Format Version 12.00

Which seems to be the most recent version.

  [1]: https://docs.microsoft.com/en-us/visualstudio/extensibility/internals/solution-dot-sln-file?view=vs-2019&redirectedfrom=MSDN#definitions-1